src/components/Announcements/Announcements.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC, useState } from 'react'
+
+import flag from 'cozy-flags'
+
+import { AnnouncementsDialog } from './AnnouncementsDialog'
+import { useAnnouncements } from 'hooks/useAnnouncements'
+import { AnnouncementsConfigFlag } from './types'
+import { useAnnouncementsSettings } from 'hooks/useAnnouncementsSettings'
+
+const Announcements: FC = () => {
+  const config = flag<AnnouncementsConfigFlag>('home.announcements')
+  const [hasBeenDismissed, setBeenDismissed] = useState(false)
+  const { values, save } = useAnnouncementsSettings()
+
+  const handleDismiss = (): void => {
+    save({
+      dismissedAt: new Date().toISOString()
+    })
+    setBeenDismissed(true)
+  }
+
+  const moreThan = config?.delayAfterDismiss ?? 1000 * 60 * 60 * 24 // 24 hours
+  const hasBeenDismissedForMoreThan = values?.dismissedAt
+    ? new Date(values?.dismissedAt) < new Date(new Date().getTime() - moreThan)
+    : true
```

Comment:
nit: Date-fns is included in the project so maybe we can use `differenceInDays` of `differenceInHours` here

Reply:
I chose to use `differenceInHours` to make it easier for the product team to test with the feature flag